vsce/server/src/server.ts

```diff
@@ -356,12 +363,16 @@ async function validateTextDocument(textDocument: TextDocument): Promise<void> {
 	connection.sendDiagnostics({ uri: textDocument.uri, diagnostics });
 
 	function addDiagnostic(element: ErrorLocation, message: string, details: string, severity: DiagnosticSeverity, code: string | undefined, suggestions: string[]) {
+		const href = `https://docs.reach.sh/${code}`;
```

Comment:
I don't know why you don't need `.html`?

Reply:
That's interesting. It appears `.html` isn't required.

Reply:
5e5061c adds it just in case we need it.